src/components/common/Slider.js

```diff
@@ -0,0 +1,54 @@
+import React,{ useState, useEffect } from 'react';
+import styled from 'styled-components';
+
+
+const ImageSlider = styled.img.attrs(props => ({src: props.src}))`
+    width: 100%;
+		min-height: 500px;
+`;
+const Arrow = styled.i`
+	border: solid black;
+	border-width: 0 3px 3px 0;
+	display: inline-block;
+	padding: 3px;
+	cursor: pointer;
+`;
+
+const LeftArrow = styled(Arrow)`
+	transform: rotate(-45deg);
+	-webkit-transform: rotate(-45deg);
+`;
+
+const RightArrow = styled(Arrow)`
+	transform: rotate(135deg);
+	-webkit-transform: rotate(135deg);
+`;
+
+const Wrapper = styled.div`
+	display: flex;
+	justify-content: center;
+	align-items: center;
+
+ ${ImageSlider} {
+	 width: 50%;
+ }
+`;
+
+const Slider = ({
+	onLeftClicked,
+	onRightClicked,
+	items = [],
+}) => {
+	const [counter, setCounter] = useState(0);
+	return (
+		<Wrapper>
+			<RightArrow onClick={() => setCounter(((counter - 1) + items.length) % items.length)}/>
+			{items.map((item,index) => (
+			counter === index && <ImageSlider src={item.data.main_image.url} />
+			))}
+			<LeftArrow onClick={() => setCounter((counter + 1)% items.length)} />
+		</Wrapper>
+	)
+}
+
+export default Slider;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏
![image](https://user-images.githubusercontent.com/26983421/164110273-ab39813f-a758-4808-a55a-899d9c3957dd.png)
On bigger screen size, the slider content is not fully centered